### DISCUSSION TITLE
Elección del tema del proyecto

### DISCUSSION THREAD

**Discussion by CarlosDev0** [rramosp/ai4eng.v1#100] [00 - General]
Hola Profesor En mi equipo estamos analizando estas tres opciones para el proyecto:  

[https://www.kaggle.com/c/petfinder-pawpularity-score/data](https://www.kaggle.com/c/petfinder-pawpularity-score/data)

[https://www.kaggle.com/debasisdotcom/parkinson-disease-detection](https://www.kaggle.com/debasisdotcom/parkinson-disease-detection)

[https://www.kaggle.com/c/digit-recognizer](https://www.kaggle.com/c/digit-recognizer): En este en particular vimos algo que nos dejó preocupados y es que el "test" no tiene la variable a estimar "label"

Por favor nos ayuda con sus comentarios para decidir por alguno de ellos.

**Comment by efsiul** [comment]
Buenas tardes, 

Quisiera saber si hay posibilidad de trabajar de forma individual,  por el siguiente motivo: Actualmente trabajo en el Laboratorio de Cocreación para la innovación en salud del Hospital General de Medellín como desarrollador de software, y desde allí tenemos un proyecto que quisiera sacar adelante con relación al procesamiento de imágenes de heridas, posteriormente se aplicaría IA para predecir diferentes posibilidades. El inconveniente que tengo es que, por ser una entidad pública, no puedo compartir información con terceros, y por ello se me hace difícil trabajar con compañeros del grupo.

**Reply by Felipe-RA** [reply]
Saludos efsiul

Adelante, puedes trabajarlo solo.

PD: La próxima vez crea un tema aparte, este es de otra persona.
